portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,25 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&family=Nunito&display=swap" rel="stylesheet">
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div id="header">
+            <div id="left"><img src="/images/profile.jpg" id="profile"></div>
+            <div id="right">
+                <h1>Hi, I'm Keely!</h1>
+                <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and 
```

Comment:
Indentation in this paragraph block looks a little inconsistent.

Reply:
Should I be putting the img tag on a new line?

Reply:
I think it would help with readability, seems like a good idea.